static/index.html

```diff
@@ -3,19 +3,51 @@
     <head>
         <meta charset="utf-8">
         <title>Zig Playground</title>
-        <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-eOJMYsd53ii+scO/bJGFsiCZc+5NDVN2yr8+0RDqr0Ql0h+rP48ckxlpbzKgwra6" crossorigin="anonymous">
+        <link rel="shortcut icon" type="image/x-icon" href="/logo.ico" />
+        <link href="/style.css" rel="stylesheet" type="text/css"/>
```

Comment:
@gsquire Do you want me to put the styles inline?
Advantages:
* Fewer requests
* Since it's a small page it's unnecessary to have multiple files
Disadvantages:
* Less clear maybe? Means index.html has more stuff going on in it

Reply:
I'm fine with having them inline. If this proves to be unwieldy in the future, we can always adjust it.